custom_components/mail_and_packages/const.py

```diff
@@ -92,6 +92,9 @@
     "Geliefert:",
     "Livré",
 ]
+AMAZON_DELIVERING_SUBJECT = [
```

Comment:
I've never seen an email from Amazon with this subject.

Reply:
in german it's "Zustellung heute: <ProductName>"

Reply:
Again since the shipper isn't actually Amazon, we wouldn't utilize this.

Reply:
I just received an email with this subject.  See: https://github.com/moralmunky/Home-Assistant-Mail-And-Packages/discussions/873